presentation/common/src/presentation-common/KeySet.ts

```diff
@@ -198,9 +205,12 @@ export class KeySet {
       if (Key.isEntityProps(value)) {
         this.add({ className: value.classFullName, id: Id64.fromJSON(value.id) } as InstanceKey);
       } else if (Key.isInstanceKey(value)) {
-        if (!this._instanceKeys.has(value.className))
-          this._instanceKeys.set(value.className, new Set());
-        this._instanceKeys.get(value.className)!.add(value.id);
+        if (!this._instanceKeys.has(value.className.toLowerCase())) {
+          this._instanceKeys.set(value.className.toLowerCase(), new Set());
+          this._lowerCaseMap.set(value.className.toLowerCase(), value.className);
+        }
+        this._lowerCaseMap.set(value.className.toLowerCase(), value.className);
```

Comment:
Could convert the class name to lower case only once.

Reply:
Fixed it